app/views/users/_response_template.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="<%= "github-repo-row-featured" if response_template.id == params[:id].to_i %> mb-4">
```

Comment:
There's some a11y improvements we could make in this partial, but none of the issues are new with this PR so I'll just quietly note them down for later 🙂